code/client/src/sdk/ue/game/traffic/c_streaming_traffic_module.h

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+#include "../../../c_ticked_module.h"
+
+#include "../../../patterns.h"
+
+namespace SDK {
+    namespace ue::game::traffic {
+        class C_StreamingTrafficModule: public C_TickedModule {
+          private:
```

Comment:
@Segfaultd I set the fields to private arbitrarily (and it works) but is that ok?

Reply:
It is ok on a technical POV but since all the SDK have no private member, let's stick to public :-)